components/NavigationDrawer/tests/snapshot/MDCBottomDrawerControllerSnapshotTests.m

```diff
@@ -152,4 +152,30 @@ - (void)testPresentedDrawerWithColoredViewsWithVerticalSizeClassCompact {
   [self generateSnapshotAndVerifyForView:self.bottomDrawerViewController.view];
 }
 
+- (void)testPresentedDrawerWithShadowElevation {
+  // Given
+  self.presentingViewController.view.frame = CGRectMake(0, 0, 375, 667);
+  self.containerViewController = [[FakeBottomDrawerContainerViewController alloc]
+      initWithOriginalPresentingViewController:self.presentingViewController
+                            trackingScrollView:nil];
+  self.containerViewController.contentViewController =
+      self.bottomDrawerViewController.contentViewController;
+  self.containerViewController.headerViewController =
+      self.bottomDrawerViewController.headerViewController;
+  self.containerViewController.contentViewController.view.backgroundColor = UIColor.whiteColor;
+  self.containerViewController.headerViewController.view.backgroundColor = UIColor.whiteColor;
+  self.containerViewController.drawerShadowColor = UIColor.blueColor;
```

Comment:
I believe the `shadowColor` and `elevation` property assignments should be the "When" clause for this test.

Reply:
Done.